### ISSUE TITLE
Deploy Code to Grails Plug-in Portal

### ISSUE THREAD

**Issue by RGBz** [davidmarquis/grails-lesscss#7]
Hi David,

Can you deploy the latest code to the Grails plug-in portal?

Thanks,
Anton

**Comment by 000panther** [comment]
Yeah, I also just tried upgrading my app, and the plugin had some problems (but it worked with the code from the Git Repository). Your plugin would work together with spring security (not like the lesscss-resources plugin, that has quite some hassles). I would do it myself, but I do assume I won't have the permissions, and the knowledge ...

**Comment by antonbayer** [comment]
i added your plugin to my grails project and i also integrated the latest twitter bootstrap less version. but i got build errors while building the war file because the plugin's version of less and lesscss-engine can't compile the bootstrap syntax.
my solution: i updated/replaced the less version to 1.3.1 and lesscss-engine to 1.3.0 and everything is running fine.
could anybody from the developer team update the version to support the latest version (eg of twitter bootstrap) and deploy the version to the grails plugin-portal?
